flows/codelingo/review/review/cli.go

```diff
@@ -222,3 +237,56 @@ func reviewAction(cliCtx *cli.Context) (chan proto.Message, <-chan *flowutil.Use
 	resultc, userVarc, errc, err := RequestReview(ctx, req, insecure)
 	return resultc, userVarc, errc, cancel, errors.Trace(err)
 }
+
+func findPath() (string, error) {
+	path := ""
+
+	// Find the root of repo
+	cmd := exec.Command("git", "rev-parse", "--show-toplevel")
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		return "", errors.Trace(err)
+	}
+	repoRoot := strings.TrimSpace(string(output))
+
+	// Check if the repo is a go project
+	goProject, err := isGoProject(repoRoot)
+	if err != nil {
+		return "", errors.Trace(err)
+	}
+
+	// Check if GOPATH is set
+	goPath := os.Getenv("GOPATH")
+
+	// Check if repo is in GOPATH
+	if goPath != "" && goProject {
+		if strings.HasPrefix(repoRoot, goPath+"/src/github.com") {
+			path = strings.TrimPrefix(repoRoot, goPath+"/src/")
+
+		} else if strings.HasPrefix(repoRoot, goPath+"/src/") {
+			path = strings.TrimPrefix(repoRoot, goPath+"/src/")
+		}
+	}
+	return path, nil
+}
+
+func isGoProject(currentDir string) (bool, error) {
+	isGo := false
+	items, err := ioutil.ReadDir(currentDir)
+	if err != nil {
+		return false, errors.Trace(err)
+	}
+	for _, item := range items {
+		if !item.IsDir() && filepath.Ext(item.Name()) == ".go" {
+			return true, nil
+
+		} else if item.IsDir() && !strings.HasPrefix(item.Name(), ".") {
+			isGo, err = isGoProject(filepath.Join(currentDir, item.Name()))
+			if err != nil {
+				return false, errors.Trace(err)
+			}
+
```

Comment:
```suggestion

			if isGo {
				return true
			}
```

Reply:
This way we return as soon as we see a go file, rather than continuing to search the rest of its siblings.